src/org/usfirst/frc4904/robot/humaninterface/drivers/DefaultDriver.java

```diff
@@ -11,7 +15,17 @@ public DefaultDriver() {
 	}
 	
 	@Override
-	public void bindCommands() {}
+	public void bindCommands() {
+		Command normalDrive = new ChassisMove(RobotMap.Component.chassis, this);
+		RobotMap.HumanInput.Driver.xbox.dPad.up.whenPressed(new ChassisTurnAbsolute(RobotMap.Component.chassis, 180, RobotMap.Component.navx, RobotMap.Component.chassisMC));
+		RobotMap.HumanInput.Driver.xbox.dPad.up.whenReleased(normalDrive);
```

Comment:
Having four of these isn't an issue since the XboxController class can't ever indicate more than one DPad button is pressed.